cmake/templates/python_win32_wrapper.cpp.in

```diff
@@ -87,20 +92,49 @@ int main(int argc, char* argv[]) try
         fprintf(stderr, "[DEBUG] Python script name: %s\n", scriptName.c_str());
 #endif
 
-        // use quoted string to ensure the correct path is used
-        return "\"" + scriptName + "\"";
+        return scriptName;
     };
 
-    const auto GetPythonExecutable = []() -> std::string
+    const auto GetPythonExecutable = [](const std::string& exeFullPath) -> std::string
     {
-        std::string pythonExecutable = "@PYTHON_EXECUTABLE@";
+        std::string scriptName;
+        scriptName += dirnameOf(exeFullPath);
+        scriptName += "\\@ARG_SCRIPT_NAME@";
+
+        std::ifstream infile(scriptName);
+
+        std::string firstLine;
+        if (std::getline(infile, firstLine) && firstLine.find(SHEBANG) == 0)
+        {
+            firstLine.erase(0, SHEBANG.size());
+
+            // trim extra spaces.
+            auto start = firstLine.find_first_not_of(" ");
+            if (std::string::npos != start)
+            {
+                firstLine.erase(0, start);
+            }
```

Comment:
What about the `else` case? I would assume that would be problem which should be handled somehow explicitly. Especially since the following condition `!firstLine.empty()` wouldn't be triggered.

Reply:
Thanks for the feedback! I reiterate this logic to do left & right trim for both spaces and tabs. That should help eliminate the problem that an ill-formed command line being treated as the Python interpreter.